utils/oscap-remediate

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2021 Red Hat Inc., Durham, North Carolina.
+#
+# Evgenii Kolesnikov <ekolesni@redhat.com>
+#
+# This is OpenSCAP's offline system remediation service wrapper script and
+# it is supposed to be executed by systemd and it is not sutable for manual
+# invocation.
+
+
+UCHAR_MAX=255
+SYSTEMD_SYSTEM_UPDATE_LINK="/system-update"
+OSCAP_REMEDIATE_CONF_BASENAME="oscap-remediate-offline.conf.sh"
+
+
+display() {
+	# Plymouth does not like messages longer than UCHAR_MAX
+	plymouth display-message --text="${1:0:${UCHAR_MAX}}"
+}
+
+log() {
+	echo "${1}"
+}
+
+die() {
+	log "${1}" >&2
+	exit 1
+}
+
+
+config=$(readlink -f "${SYSTEMD_SYSTEM_UPDATE_LINK}")
+
+if [[ "$(basename ${config})" != "${OSCAP_REMEDIATE_CONF_BASENAME}" ]]; then
+	log "The ${SYSTEMD_SYSTEM_UPDATE_LINK} symlink does not point to oscap offline remediation configuration, ignoring"
+	exit 0
+fi
+
+rm -f "${SYSTEMD_SYSTEM_UPDATE_LINK}"
+log "Removed ${SYSTEMD_SYSTEM_UPDATE_LINK}"
+log "Found config: ${config}"
+
+source "$config"
```

Comment:
@evgenyz  We will need to have the format of the config file documented somewhere - ie. document which variables the service accepts and what is their meaning so that the people/software using this service know how to set this service up. Ideally in this service's documentation.

Reply:
Done